apps/charterafrica/.env

```diff
@@ -1,7 +0,0 @@
-NEXT_PUBLIC_APP_LOGO_URL=https://charter.dev.codeforafrica.org/logos/main.png
```

Comment:
[In Next.js world](https://nextjs.org/docs/basic-features/environment-variables), `.env` file should be committed to git containing public vars/non-secrets. They encourage the use of `.env.local` file for secrets and this is the file that should **never** be committed to git.